yarn.lock

```diff
@@ -1166,10 +1166,6 @@ es5-ext@^0.10.7, es5-ext@^0.10.8, es5-ext@~0.10.11, es5-ext@~0.10.2, es5-ext@~0.
     es6-iterator "2"
```

Comment:
Yarn did this, guess it's also updating other things that weren't included in my Yarn PR on release.